pkg/autodiscovery/telemetry/telemetry.go

```diff
@@ -60,6 +60,6 @@ var (
 		"Poll duration distribution by config provider (in seconds).",
 		// The default prometheus buckets are adapted to measure response time of network services
 		prometheus.DefBuckets,
-		telemetry.Options{NoDoubleUnderscoreSep: true},
+		telemetry.Options{NoDoubleUnderscoreSep: true, NativeHistogramZeroThreshold: -0.1},
```

Comment:
Does this actually have an effect? I don't think we're using native histograms.

Reply:
Indeed, apparently the issue could be in the aggregator. Will open a different PR